util/Cargo.toml

```diff
@@ -1,11 +1,11 @@
 [package]
 name = "epic_wallet_util"
-version = "3.0.0"
+version = "3.3.2"
 authors = ["Epic Developers <mimblewimble@lists.launchpad.net>"]
 description = "Util, for generic utilities and to re-export epic crates"
 license = "Apache-2.0"
 repository = "https://github.com/mimblewimble/epic-wallet"
```

Comment:
Change this repository to "https://github.com/EpicCash/epic-wallet"

Reply:
Done